src/holon/rule_engine/repositories/repository_base.py

```diff
@@ -129,3 +147,19 @@ def remove(self, object):
         <CHANGES INTERNAL STATE>
         """
         raise NotImplementedError()
+
+
+def model_base_type(object: PolymorphicModel) -> type:
```

Comment:
Als het goed is dit een standaard functie binnen Polymorphic, namelijk `get_base_polymorphic_model`: https://django-polymorphic.readthedocs.io/en/stable/api/polymorphic.utils.html#polymorphic.utils.get_base_polymorphic_model

Als deze het zelfde doet als deze functie zou ik aanraden de versie van polymorphic te gebruiken.

Reply:
Done.